Support pkix combined

karel-harjono opened the pull request:
- [ ] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [ ] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [ ] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

SOE build with PKIX: https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=4f174979-dd93-4f7d-9298-fe9f6463a916&tab=Steps

wrodrig committed:
Support PKIX KeyManagerFactory for client and server aliases

tloodu committed:
Update error case when alias is not resolved from list to return prefixed client alias

utle committed:
upgrade unboundid to version 7.0.3

tloodu committed:
Remove null return in PKIX chooseClientAlias

karel-harjono commented:
!build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=bcf1aa8e-9d7e-43b7-b097-f1802e472f34) - :warning: completed with warnings.)

_Note: Target locations of links might be accessible only to IBM employees._

karel-harjono committed:
Merge branch 'integration' into support-pkix-combined

karel-harjono commented:
!build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=d18e4163-faab-4f45-a647-ff2e9336b020) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

utle commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=b82ef3e8-2b68-4456-b9c5-f9531a2c13a6) - :x: completed with errors/failures.)
fat.buckets.to.run=all
fat.test.mode=full
spawn.zos=true

_Note: Target locations of links might be accessible only to IBM employees._

karel-harjono commented:
SOE build with PKIX: https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=4f174979-dd93-4f7d-9298-fe9f6463a916&tab=Steps

karel-harjono commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=0fd1b191-447a-4bb8-b026-bb7ebe4bfd42) - :x: completed with errors/failures.)
fat.buckets.to.run=all
fat.test.mode=full
spawn.zos=true

_Note: Target locations of links might be accessible only to IBM employees._

karel-harjono committed:
Revert "Remove null return in PKIX chooseClientAlias"

This reverts commit 630be5c2f1b1b3c5177868456510f94e314bbc49.

karel-harjono commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=9fa6d22d-5365-449a-a175-97841432944a) - :x: completed with errors/failures.)
fat.buckets.to.run=all
fat.test.mode=full
spawn.zos=true

_Note: Target locations of links might be accessible only to IBM employees._

karel-harjono committed:
upload WrapperKeyManager.java unboundid src

karel-harjono committed:
update inboundid in memory ldap to support PKIX alias prefix

rangaran committed:
updated keystore values (#8)

Fixed tests

Revert "update inboundid in memory ldap to support PKIX alias prefix"

This reverts commit 7ed850b1a514315f4eadc6cb68a65139b2b55ae5.

Co-authored-by: Karel Harjono <karel.harjono@ibm.com>
